docs/source/install/overview.rst

```diff
@@ -0,0 +1,62 @@
+Overview: Single-box Reference Deployment
+=========================================
+
+.. todo:: short intro paragraph: this is reference deployment for single box, follow installation
+    guide to get here.
+
+.. figure:: https://docs.google.com/drawings/d/1X6u8BB9bnWkW8C81ERBvjIKRfo9mDos4XEKeDv6YiF0/pub?w=960&amp;h=720
```

Comment:
I just don't think we should make this first impression on potential users. It's a good pic when you need to explain deployment to power users or enterprise customers who know their stuff well and played with St2 for a while, but it's arguably not very good for newcomers, so maybe it shouldn't be here. @dzimine: thoughts?